src/main/java/org/prebid/server/bidder/sovrn/SovrnBidder.java

```diff
@@ -114,6 +118,14 @@ private String resolveTagId(ExtImpSovrn sovrnExt) {
         return tagId;
     }
 
+    private ObjectNode resolveImpExt(ExtImpSovrn sovrnExt, ObjectNode impExt) {
+        final ObjectNode sovrnImpExt = impExt.deepCopy();
+        sovrnImpExt.remove(EXT_BIDDER_PARAM);
```

Comment:
Why are we removing `bidder` here?

Reply:
I tried comparing it to Blackbird's openrtb-sovrn.proto:

https://github.com/sovrn/blackbird/blob/23bc2e065166a5c705d9b1f7d47fc03dbb7b59d3/blackbird-openrtb-ext/src/main/proto/openrtb-sovrn.proto#L26C1-L34C1

there is no bidder field handled there, so I thought it was redundant. Please correct me if I am wrong.

Reply:
OK, I've figured out what was bothering me. The ExtImpSovrn class has nothing to do with our protobuf definition in Blackbird It is a local class in prebid server (org/prebid/server/proto/openrtb/ext/request/sovrn/ExtImpSovrn.java). Here's an example of the javascript that is passing into the adapter.
https://docs.prebid.org/dev-docs/bidders/sovrn.html
I believe the line `sovrnImpExt.remove(EXT_BIDDER_PARAM);` actually does nothing.

Reply:
Do tests pass with your changes?

Reply:
Of course tests pass with my changes.
Yes ExtImpSovrn is local class in prebid server, but eventually the Bidrequest generated in prebid-server gets to prebid endpoint in Blackbird and gets parsed according to proto, doesn't it?